service/src/main/java/com/epam/esm/service/impl/UserServiceImpl.java

```diff
@@ -21,7 +33,10 @@
 @Transactional
```

Comment:
I would advice not using @Transactional at a class level

Reply:
ok, thanks. I have moved them from class-level to method-level.